packages/cli/src/constructs/alert-channel.ts

```diff
@@ -54,11 +66,15 @@ export abstract class AlertChannel extends Construct {
     this.sslExpiryThreshold = props.sslExpiryThreshold
   }
 
+  static fromId (id: string|number) {
```

Comment:
@umutuzgur , I think you could specify just `number` here. The table id is a `serial`.

```suggestion
  static fromId (id: number) {
```